ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Week 7: Git Challenge
+
+Welcome to week 7 of software training! This week, you'll be completing a series of `git` and GitHub challenges. **To get started, click the "Submit new issue" at the bottom of this page!** From there, work through the rest of this checklist.
+
+If you need any help, we recommend these online resources:
+- https://help.github.com/articles/github-glossary/
+- https://try.github.io/levels/1/challenges/1
+- https://learngitbranching.js.org/
+- https://chris.beams.io/posts/git-commit/
+
+You can, of course, also flag down an instructor if you run into anything.
+
+## Your objectives
+- [ ] Star this repository
+- [ ] Make a personal copy of the repo that will live on your GitHub account. This allows you to freely make changes to the project without affecting the original. Your copy will remain associated with the original, allowing you to submit a pull request to the original author.
+- [ ] Make a copy of that repository that lives on your computer instead of on a github server somewhere. Don’t forget, this version of the repo is connected to the remote version so that changes can be synced between the two.
+- [ ] Set your forked repository as a remote named origin
+- [ ] Set our repository as a remote named rj
+- [ ] Create a branch named your gt username (ex. gburdell3)
+- [ ] Use the terminal to create a folder named “2017” in the root of the repo
+- [ ] Use the terminal to create an empty text file named yourgtusername.txt in the 2017 folder
+- [ ] Stage/commit the file (make sure you leave good commit messages)
+- [ ] Create a new branch named gtusername/add-text
```

Comment:
Where should this branch branch off from?

Reply:
I guess specifying we're branching off of master wouldn't hurt